app/models/user.rb

```diff
@@ -102,8 +102,16 @@ def pending_profile?
     pending_profile.present?
   end
 
+  def gpo_verification_pending_profile?
+    gpo_verification_pending_profile.present?
+  end
+
   def pending_profile
-    profiles.gpo_verification_pending.order(created_at: :desc).first
```

Comment:
also are we going to backfill this data? do we need to have "double-read" paths to make sure older records get handled correctly here?

Reply:
I've already done the backfill. I do need to change this to `gpo_verification_pending_at` though